resources/js/beatmap-discussions/discussions.tsx

```diff
@@ -172,10 +172,7 @@ export class Discussions extends React.Component<Props> {
     const parentDiscussion = discussion.parent_id != null ? this.props.currentDiscussions.byFilter.total.reviews[discussion.parent_id] : null;
 
     return (
-      <div
-        key={discussion.id}
-        className={`${bn}__discussion`}
-      >
+      <React.Fragment key={discussion.id}>
```

Comment:
what's this for? (instead of just returning discussion component as is)

Reply:
wait, I thought there were 2 elements there when I changed it despite saying it wasn't needed 🙈